extensions/keyboard-layout-switcher/src/toggle.ts

```diff
@@ -0,0 +1,7 @@
+import { showHUD } from "@raycast/api";
+import { LayoutManager } from "./model/LayoutManager";
+
+export default async function main() {
+  const selectedInput = await LayoutManager.setNextInput();
+  await showHUD(`${selectedInput.title} selected`);
```

Comment:
I think it might be nice to use slightly more verbose wording to tell the user what happened:

```suggestion
  await showHUD(`⌨️ Activated '${selectedInput.title}' Layout`);
```